shell/platform/darwin/ios/framework/Source/FlutterKeyboardManager.mm

```diff
@@ -89,7 +76,7 @@ - (void)handlePress:(nonnull FlutterUIPressProxy*)press
       // encounter.
       NSAssert([_primaryResponders count] >= 0, @"At least one primary responder must be added.");
 
-      __block auto weakSelf = [self getWeakNSObject];
+      __block __weak __typeof(self) weakSelf = self;
```

Comment:
Removed the `WeakNSObjectFactory` in favor of `__weak` where it's used before the block.  I can swap `__typeof(self)` to `FlutterKeyboardManager*` I don't really have a preference.